src/main/java/org/reaktivity/nukleus/tcp/internal/stream/ClientStreamFactory.java

```diff
@@ -196,9 +193,13 @@ public void doConnect(
         SocketChannel channel,
         InetSocketAddress remoteAddress,
         String acceptReplyName,
-        long correlationId)
+        long correlationId,
+        MessageConsumer outputThrottle,
+        long outputStreamId,
+        BiConsumer<MessageConsumer, Long> setCorrelatedInput)
```

Comment:
I'll make it a LongObjectBiConsumer since http, http2 and ws all define the class. How about we add it to shared core (nukleus.java)?